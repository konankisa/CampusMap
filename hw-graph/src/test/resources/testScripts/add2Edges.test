# Creates an empty graph, adds 3 nodes to it,
# and adds 2 edges connecting the 3 nodes

# Create a graph
CreateGraph graph

# Add a node
AddNode graph node1

# Add another node
AddNode graph node2

# Add a third node
AddNode graph node3

# Add an edge from node1 to node2 in graph
AddEdge graph node1 node2 edge1

# Add another edge from node2 to node3 in graph
AddEdge graph node2 node3 edge2