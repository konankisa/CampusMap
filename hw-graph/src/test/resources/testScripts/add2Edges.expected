# Creates an empty graph, adds 3 nodes to it,
# and adds 2 edges connecting the 3 nodes

# Create a graph
created graph graph

# Add a node
added node node1 to graph

# Add another node
added node node2 to graph

# Add a third node
added node node3 to graph

# Add an edge from node1 to node2 in graph
added edge edge1 from node1 to node2 in graph

# Add another edge from node2 to node3 in graph
added edge edge2 from node2 to node3 in graph